modules/express/README.md

```diff
@@ -163,31 +163,48 @@ BitGo Express currently supports the following proxy protocols:
 * SOCKSv5
 * PAC
 
+### External Signing Mode
+BitGo Express can be run in an external signing mode, where the signing of transactions is performed in a separate instance of BitGo Express that has access to the private keys. This may be preferable for users who would like to apply their signature to their transactions using a more secure environment than BitGo SDK or BitGo Express, such as an offline signer.
```

Comment:
Does this allow for the signer to be offline?

It might be more accurate to say a signer with restricted access or network connectivity.